packages/core/src/tests/uma.test.ts

```diff
@@ -38,6 +39,7 @@ const generateKeypair = async () => {
     publicKey,
   };
 };
+const oneWeekAgo = Date.now() - 1000 * 60 * 60 * 24 * 7;
```

Comment:
nit: This should technically maybe be a function? Otherwise it'll be not actually a week ago when references.